src/lib/utils.test.ts

```
@@ -0,0 +1,174 @@
+import { PLAN_LIMITS } from '@/constants'
+import { PlanLimits } from '@/types'
```

ESLint

[error] All imports in the declaration are only used as types. Use `import type`.

(`@typescript-eslint/consistent-type-imports`)

---

[error] `vitest` import should occur before import of `@/constants`

(import/order)